src/winuwp/project_version

```diff
@@ -0,0 +1 @@
+0
```

Comment:
Here and a few other places: add a trailing newline at the end of these files.

Reply:
fixed in next commit

Reply:
which other places? ;-)